shopfloor_full_location_reservation/services/zone_picking.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2023 Michael Tietz (MT Software) <mtietz@mt-software.de>
+# Copyright 2023 Jacques-Etienne Baudoux (BCIM) <je@bcim.be>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo.addons.component.core import Component
+
+
+class ZonePicking(Component):
+    _inherit = "shopfloor.zone.picking"
+
+    def _handle_full_location_package_reservation(self, package):
+        return package and self.work.menu.full_location_reservation
```

Comment:
It is not a full location reservation, it is a full package reservation.

If you had scanned a product not part of a package, it should trigger a full product reservation. Same for lot.